### PR TITLE
Look in same directory when 'use'-ing. Also tests

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -73,6 +73,13 @@ impl Default for Args {
     }
 }
 
+pub fn path_to_module(current_file: &Path, module: &str) -> PathBuf {
```
Comment (FredTheDino) [inline src/lib.rs:76]: Do you see this being used somewhere else? Why is it in lib?

File: progs/tests/import/main.sy [hunk 2]
```diff
@@ -0,0 +1,6 @@
+use _other
```
Comment (FredTheDino) [inline progs/tests/import/main.sy:1]: It would be nice with a test with circular dependencies, just to make sure.

File: src/sectionizer.rs [hunk 3]
```diff
@@ -1,9 +1,11 @@
 use crate::error::{Error, ErrorKind};
 use crate::tokenizer::{PlacedToken, Token, file_to_tokens};
+use crate::path_to_module;
 
 use std::collections::HashSet;
 use std::path::{Path, PathBuf};
 
+#[derive(Debug)]
```
Comment (FredTheDino) [inline src/sectionizer.rs:8]: :+1:

**Review by FredTheDino — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (sornas) [reply src/lib.rs:76]:
```
$ rg path_to_module src/**
src/sectionizer.rs
3:use crate::path_to_module;
54:                    let use_file = path_to_module(&path, file);

src/lib.rs
76:pub fn path_to_module(current_file: &Path, module: &str) -> PathBuf {

src/compiler.rs
6:use crate::{Blob, Block, Op, Prog, RustFunction, Type, Value, path_to_module};
1783:                    let path = path_to_module(file, &name);
```

**Review by FredTheDino — APPROVED** [review]
LGTM

**Pull request merged**